### ISSUE TITLE
Enable page specific URLs

### ISSUE THREAD

**Issue by Zasco** [memo33/sc4pac-gui#13]
Currently, every page has the `localhost:51515/webapp/` URL. The expected behaviour would be to have page specific URLs like `localhost:51515/webapp/jasoncw:cecil-hotel`. This would allow for direct linking.

**Comment by memo33** [comment]
Those links to `localhost:51515` would be a bit useless to anyone without the web-app. I imagine most people would use the desktop app.

Instead, I'm planning to add a button that links to the package on the website, such as https://memo33.github.io/sc4pac/channel/?pkg=memo:submenus-dll. That link is accessible to everyone, so can be shared.

**Comment by Zasco** [comment]
> Those links to `localhost:51515` would be a bit useless to anyone without the web-app. I imagine most people would use the desktop app.

Of course, I thought about it and to change the message accordingly only later. I though it's what you where somehow doing here:
https://github.com/memo33/sc4pac-tools/pull/21/files#diff-0ddf5ab84f43d4534516d3a3694e47a3e54f8cb7825d215a323d18340ecf93c0R142-R161
I understand this would however open whichever version of the GUI you have.
Although only used by the web-app, this would improve it's navigation and **UX**. With this feature, there would be no need to navigate to a specific page, one could only paste it's URL to reach it directly.

**Comment by memo33** [comment]
> I understand this would however open whichever version of the GUI you have.

It doesn't open the GUI. The GUI must already be open. Instead, it tells the GUI to navigate to that package. Simply pasting that URL could work for the web-app, but not for the desktop app, as the HTTP request from that code snippet is a POST. Possibly, the API could serve a dummy page in that case with a button to trigger the desktop app, but this feels like a kludge solving a self-inflicted problem.

Ideally, we'd avoid external references to `localhost:51515` altogether to avoid making blind assumptions about what port the app is running on. This could be solved with a custom `sc4pac://…` URI scheme, but I don't know how to register that with the OS, specifically for the web-app launcher scripts on Windows and macOS.

**Comment by memo33** [comment]
It'll be the last one that was opened. (Not that opening multiple connections is really intended usage – it's a web-app, not a website.)
